reference/api/documents.mdx

```diff
@@ -4,31 +4,113 @@ The `/documents` route allows you to create, manage, and delete documents.
+
+Get a set of documents.
+
+Using `offset` and `limit`, you can browse through all your documents.
+
+<Capsule intent="warning">
+`filter` will not work without first explicitly adding attributes to the [`filterableAttributes` list](/reference/api/settingss#update-filterable-attributes). [Learn more about filters in our dedicated guide.](/learn/advanced/filtering#filter-basics)
+</Capsule>
+
+### Path parameters
+
+| Name              | Type   | Description                                                            |
+| :---------------- | :----- | :--------------------------------------------------------------------- |
+| **`index_uid`** * | String | [`uid`](/learn/core_concepts/indexes#index-uid) of the requested index |
+
+### Body
+
+| Name         | Type                                    | Default Value | Description                                                           |
+| :----------- | :-------------------------------------- | :------------ | :-------------------------------------------------------------------- |
+| **`offset`** | Integer                                 | `0`           | Number of documents to skip                                           |
+| **`limit`**  | Integer                                 | `20`          | Number of documents to return                                         |
+| **`fields`** | Array of strings/`null`                 | `*`           | Document attributes to show (case-sensitive, comma-separated)         |
+| **`filter`** | String/Array of array of strings/`null` | N/A           | Refine results based on attributes in the `filterableAttributes` list |
+
+<Capsule intent="note">
+Sending an empty payload (`--data-binary '{}'`) will return all documents in the index.
+</Capsule>
+
+### Response
+
+| Name          | Type    | Description                            |
+| :------------ | :------ | :------------------------------------- |
+| **`results`** | Array   | An array of documents                  |
+| **`offset`**  | Integer | Number of documents skipped            |
+| **`limit`**   | Integer | Number of documents returned           |
+| **`total`**   | Integer | Total number of documents in the index |
+
+### Example
+
+<CodeSamples id="get_documents_post_1" />
+
+#### Response: `200 Ok`
+
+```json
+{
+  "results":[
+    {
+      "title":"The Travels of Ibn Battuta",
+      "genres":[
+        "Travel",
+        "Adventure"
+      ],
+      "language":"English",
+      "rating":4.5
+    },
+    {
+      "title":"Pride and Prejudice",
+      "genres":[
+        "Classics",
+        "Fiction",
+        "Romance",
+        "Literature"
+      ],
+      "language":"English",
+      "rating":4
+    },
+    …
+  ],
+  "offset":0,
+  "limit":3,
+  "total":5
+}
+```
+
+## Get documents with GET
+
+<Capsule intent="danger">
+We recommend using POST `/indexes/{index_uid}/documents/fetch` as this endpoint will be deprecated in the near future.
```

Comment:
Let's reword this so we don't use "we".

Reply:
Is there a reason for not using it? We use it in [warning boxes](https://www.meilisearch.com/docs/reference/api/settings#:~:text=the%20returned%20documents.-,WARNING,-Updating%20distinct%20attributes) in the settings API reference

Reply:
I thought we had agreed to slowly reduce usage of first person (singular and plural) across our docs. Other companies, like Microsoft, advise [avoiding whenever possible](https://learn.microsoft.com/en-us/style-guide/grammar/person#avoid-plural-first-person-we-us).

It's also good practice, as rephrasing "we recommend [doing X]" often results in the text being more to the point:


> We recommend that you change your password
vs
> Change your password 

> We recommend that you use the default schedule setting to keep your software up to date.
vs
> Keep your software up to date with the default schedule setting.

> We recommend using POST `/indexes/{index_uid}/documents/fetch` as this endpoint will be deprecated in the near future.
vs
> This endpoint will be deprecated in the near future. Consider using `/indexes/{index_uid}/documents/fetch` instead.
or even
> Meilisearch will deprecate this endpoint in the near future. Use `/indexes/{index_uid}/documents/fetch` instead.